DataConnectors/WorkplaceFacebook/WorkplaceWebhooksTrigger/__init__.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import hashlib
+import hmac
+import os
+import azure.functions as func
+import json
+import re
+import base64
+import requests
+import datetime
+
+
+AppSecret = os.environ['WorkplaceAppSecret']

```

Comment:
In order to work azure function correctly , Could you please update following environment variable names as per azure deploy.
1. Replace "WorkplaceAppSecret" with "WorkplaceForFacebookAppSecret"
2. Replace  "WorkplaceVerifyToken" with "WorkplaceForFacebookVerifyToken"

Reply:
Sure, I`ll update it

Reply:
@averbn , I have pushed those changes can you please check once. Thanks !